### PR TITLE
Add office hours events

### PR DESCRIPTION
**Pull request by phumberdroz**
**Which issue(s) this PR fixes**:

Add Office Hours Events to the sigs yaml

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -1194,6 +1194,18 @@ sigs:
       tz: PT (Pacific Time)
       frequency: weekly
       url: https://docs.google.com/document/d/1oLXv5_rM4f645jlXym_Vd7AUq7x6DV-O87E6tcW1sjU/edit
+    - description: Office Hours European Edition
```
Comment (parispittman) [inline sigs.yaml:1197]: could we make it clear in the description that this is for end users and not q&a for upstream? thanks~

Comment (phumberdroz) [reply sigs.yaml:1197]: Would something like `Community Office Hours European Edition` sound better?

Thoughts @castrojo ?

File: sig-contributor-experience/README.md [hunk 2]
```diff
@@ -88,8 +88,8 @@ Creates and runs contributor-focused events, such as the Contributor Summit.  Ev
   - Slack: [#events](https://kubernetes.slack.com/messages/events)
 - **Meetings:**
   - Events Planning Subproject: [Mondays at 10:00 PT (Pacific Time)](https://docs.google.com/document/d/1oLXv5_rM4f645jlXym_Vd7AUq7x6DV-O87E6tcW1sjU/edit) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00&tz=PT%20%28Pacific%20Time%29).
-  - Office Hours European Edition: [Wednesdays at 09:00 ET (Eastern Time)](https://hackmd.io/@k8s/office-hours) (monthly on 2th Wednesday). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=09:00&tz=ET%20%28Eastern%20Time%29).
-  - Office Hours Western Edition: [Wednesdays at 12:00 ET (Eastern Time)](https://hackmd.io/@k8s/office-hours) (monthly on 2th Wednesday). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=12:00&tz=ET%20%28Eastern%20Time%29).
+  - Office Hours European Edition (Open Q&A for kubernetes related questions): [Wednesdays at 09:00 ET (Eastern Time)](https://hackmd.io/@k8s/office-hours) (monthly on 3rd Wednesday). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=09:00&tz=ET%20%28Eastern%20Time%29).
```
Comment (castrojo) [inline sig-contributor-experience/README.md:91]: Can we do `Open Q&A for end-user kubernetes related questions` to make the end user part more explicit? Thanks!

**Comment by phumberdroz** [comment]
Wuhu first PR merged 🥳 

Thanks to you all!

**Comment by nikhita** [comment]
@phumberdroz Congratulations!!! :tada: :tada: :tada: 

/woof

**Pull request merged**